### PR TITLE
Update without notice screens

### PR DESCRIPTION
**Pull request by Davidslv**
Story: https://mojdigital.teamwork.com/#/tasks/20353649

Screenshot 2020-05-01 at 13 31 12
Screenshot 2020-05-01 at 13 31 30

### CODE REVIEW

File: app/views/steps/application/without_notice_details/edit.html.erb [hunk 1]
```diff
@@ -1,33 +1,28 @@
...
-        f.radio_button_fieldset :without_notice_frustrate, inline: true do |fieldset|
-          fieldset.radio_input(GenericYesNo::YES, panel_id: :without_notice_frustrate_panel)
-          fieldset.radio_input(GenericYesNo::NO)
-          fieldset.revealing_panel(:without_notice_frustrate_panel) do |panel|
-            panel.text_area :without_notice_frustrate_details, size: '40x4', class: 'form-control-3-4'
-          end
-        end
-      %>
+      <%= f.govuk_radio_buttons_fieldset :without_notice_frustrate, legend: { tag: 'span', size: 'm' } do %>
+        <%= f.govuk_radio_button :without_notice_frustrate, GenericYesNo::YES, link_errors: true %>
```
Comment (zheileman) [inline app/views/steps/application/without_notice_details/edit.html.erb:14]: I think this is the same issue as per my comment here:
https://github.com/ministryofjustice/c100-application/pull/900#discussion_r418512802

The text area needs to open with the YES radio option.

File: app/views/steps/application/without_notice_details/edit.html.erb [hunk 2]
```diff
@@ -1,33 +1,28 @@
...
-        f.radio_button_fieldset :without_notice_impossible, inline: true do |fieldset|
-          fieldset.radio_input(GenericYesNo::YES, panel_id: :without_notice_impossible_panel)
-          fieldset.radio_input(GenericYesNo::NO)
-          fieldset.revealing_panel(:without_notice_impossible_panel) do |panel|
-            panel.text_area :without_notice_impossible_details, size: '40x4', class: 'form-control-3-4'
-          end
-        end
-      %>
+      <%= f.govuk_radio_buttons_fieldset :without_notice_impossible, legend: { tag: 'span', size: 'm' } do %>
+        <%= f.govuk_radio_button :without_notice_impossible, GenericYesNo::YES, link_errors: true %>
```
Comment (zheileman) [inline app/views/steps/application/without_notice_details/edit.html.erb:21]: Same issue here. The text area needs to open with the YES radio option.

File: config/locales/en.yml [hunk 3]
```diff
@@ -1177,13 +1172,19 @@ en:
...
         without_notice_details: Give details of why you’re asking for a without notice hearing
         without_notice_frustrate_details: *provide_details
         without_notice_impossible_details: *provide_details
+        without_notice_frustrate_options:
+          <<: *YESNO
+        without_notice_impossible_options:
+          <<: *YESNO
       steps_application_litigation_capacity_form:
         reduced_litigation_capacity_options:
-          <<: *YESNO
```
Comment (zheileman) [inline config/locales/en.yml:1186]: Bad merge?

Comment (Davidslv) [reply config/locales/en.yml:1186]: Yep, thanks for spotting that

**Pull request merged**